common/src/main/java/com/wynntils/core/objects/TimedSet.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.objects;
```

Comment:
Move to utils/objects, I'd rather we not make a new dir

Reply:
It already existed

Reply:
?

Reply:
This is a new file?

Reply:
But the package existed

Reply:
Moved it though

Reply:
Odd, I think its only on your client

Reply:
> Moved it though

Sorry to ask again, but this is not wynn specific and would be better off in the com.wynntils.utils.objects package

Reply:
np moved